src/inverseRobot.js

```diff
@@ -7,7 +7,21 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const newRobot = {};
+  const values = Object.values(robot);
+  const keys = Object.keys(robot);
+
+  for (let i = 0; i < values.length; i++) {
```

Comment:
```suggestion

  for (const key in robot) {
```
consider to use for in loop

and make everything in one loop. 
`const value = robot[key];`